subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -58,6 +58,31 @@ Usage of invalid Java toolchain specifications is no longer supported.
 Related build errors can be avoided by making sure that language version is set on all toolchain specifications.
 See <<toolchains#sec:configuring_toolchain_specifications,user manual>> for more information.
 
+==== Changing test framework after setting test framework options is now an error
+
+When configuring the built-in test task for Java/Groovy/Scala projects, Gradle no longer allows you to
```

Comment:
```suggestion
When configuring the built-in test task for Java, Groovy, and Scala projects, Gradle no longer allows you to
```
nitpick: slashes don't feel like natural parts of a sentence, and tend to disrupt reading. Consider switching to commas.